### ISSUE TITLE
Take full schema

### ISSUE THREAD

**Issue by RangerMauve** [cainus/mongoJsonSchema#6]
I notice that this library seems to just be taking the "properties" value from a schema and not the entire schema in one go. Would it be possible to have it load a full schema to save the step of fetching properties out?

**Comment by cainus** [comment]
It would, but it gets pretty redundant, because it must always have an object at the top of the schema (because mongo docs are objects).  Also accepting any schema would mean we need error handling for schemas types that aren't an object at the top.  Know what I mean?  Does that makes sense?

**Comment by RangerMauve** [comment]
Yeah, I can see that. It's not too much hassle for me to work around, either. In my particular use-case, the full schema file gets defined for processing, and then I guess I'll take just the properties property from it when using this library.
